### DISCUSSION TITLE
No extension file exported

### DISCUSSION THREAD

**Discussion by jaxparrow07** [shreyashsaitwal/rush-cli#12] [General] [answered]
![image](https://user-images.githubusercontent.com/36300128/114270518-778d8780-9a2a-11eb-858c-f0a36ff1cf4b.png)
It shows build successful. But no output file.

**Comment by shreyashsaitwal** [comment]
Can you try building with `-rx` flag and post the output here?

**Reply by jaxparrow07** [reply]
> Can you try building with `-rx` flag and post the output here?
![image](https://user-images.githubusercontent.com/36300128/114271988-c1c63700-9a31-11eb-8851-09c27ce4d13e.png)
Still no output of the extension

**Reply by shreyashsaitwal** [reply]
Can you go to your rush installation folder, then run `chmod +x ./tools/apache-ant-1.10.9/bin/ant` and then build again?

**Reply by jaxparrow07** [reply]
![image](https://user-images.githubusercontent.com/36300128/114272795-2e8f0080-9a35-11eb-85b8-e19c4ae3949f.png)
Showing different error output.. I think there is problem with some executable permissions.

**Reply by shreyashsaitwal** [reply]
Can you see if a file named `build.xml` exists in `<rush>/tools/apache-ant-1.10.9/`.

**Reply by jaxparrow07** [reply]
Yeah

**Comment by shreyashsaitwal** [comment, marked as answer]
Fixed since v1.1.0, the latest version is 1.2.0.
